### PR TITLE
Maya/191869 pango tooltip

### PR DESCRIPTION
**Pull request by mmmmmmaya**
### Summary
- **What:** Update lineage tooltip content
- **Ticket:** [[sc-191869]](https://app.shortcut.com/genepi/story/191869)
- **Env:** https://pangott-frontend.dev.czgenepi.org/ (but dev db doesn't have new lineage data yet, IIRC)

#### Before: 
Clipboard 2022-07-04 at 11 48 19 AM

#### After: 
Screen Shot 2022-04-26 at 2 57 13 PM

### Notes
The tooltip was starting to look squished, so I opted for a wider tooltip option with the extra data. @Janeece let me know what you think.

### Checklist
- [x] I merged latest `trunk`
- [x] I manually verified the change
- [x] I added labels to my PR
- [x] I either asked design for a review or hid my changes behind a feature flag
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

File: src/frontend/src/views/Data/components/LineageTooltip/index.tsx [hunk 1]
```diff
@@ -1,31 +1,26 @@
...
 
 export const LineageTooltip = ({ lineage }: Props): JSX.Element => {
   return (
     <>
       {DISPLAY_ORDER.map((key) => {
         let value = lineage[key];
-        if (key === "confidence") {
-          value = `${value}%`;
+        if (key === "last_updated") {
+          value = `${value?.slice(0, 10)}`;
```
Comment (phoenixAja) [inline src/frontend/src/views/Data/components/LineageTooltip/index.tsx:24]: can you add a comment on why you need this slice?

**Review by phoenixAja — APPROVED** [review]
code looks good! Hard to check without rdev data.. just do a spot check when it goes on staging

File: src/frontend/src/views/Data/components/LineageTooltip/index.tsx (hunk 1 above)
Comment (danrlu) [reply src/frontend/src/views/Data/components/LineageTooltip/index.tsx:24]: Current display carries time stamp that's not useful
image

**Review by Janeece — APPROVED** [review]
Looks great, thanks for watching out for the visuals!

**Comment by danrlu** [comment]
Purely out of learning, where does the [full Pango output](https://github.com/chanzuckerberg/czgenepi/blob/8835f5a76fb01c7937a357f834012ee727cc2607/src/backend/aspen/workflows/pangolin/save.py#L53) get parsed to obtain Scorpio columns? Or should I ask Jess?

**Comment by mmmmmmaya** [comment]
@danrlu you probably want to look at #1042

**Comment by mmmmmmaya** [comment]
Here's how it looks on staging
Screen Shot 2022-04-27 at 9 03 58 AM

**Comment by danrlu** [comment]
Looks GREAT!! Thanks Maya!

**Pull request merged**